Patches/Mechanized Armor Set/Ammo_&_Apparel_Defs.xml

```diff
@@ -0,0 +1,139 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Mechanized Armor Set</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				
+				<!--Plate Helm-->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/statBases</xpath>
+					<value>
+						<Bulk>100</Bulk>
+						<WornBulk>0</WornBulk>

```

Comment:
The bulk is very high for the helmet, and it should have *some* wornBulk. I've give it 5 bulk and 1 wornBulk.

Additionally, the helmet description mentions some protection from harmful gases. I'd include:

```
  <li Class="PatchOperationAdd">
    <xpath>Defs/ThingDef[defName="Mechanized_Helm_Apparel"]/equippedStatOffsets</xpath>
    <value>
      <SmokeSensitivity>-0.8</SmokeSensitivity>
    </value>
  </li>
  ```